cpp/src/arrow/util/benchmark_util.h

```diff
@@ -19,9 +19,8 @@
 #include <cstdint>
```

Comment:
Can you revert the changes to this file?

* We like to keep 3rd party includes in their own group and not clustered with arrow includes.
* Closing namespaces don't need the `;`

Reply:
Sorry about that! Should be good here.